Yoast/wordpress-seo
# Add explanation to Media post type in configurator

## Summary

This PR can be summarized in the following changelog entry:

Add explanation to the post type "Attachment" to explain why it is in the list for sitemap entries.

## Relevant technical choices:

* Added a custom property functionality in the Post Type Factory, this is the class that handles post type fields.

## Test instructions

This PR can be tested by following these steps:

* Open the configurator and go to the "Post Type visibility" page to see the explanation under the "Media" post type.

Fixes https://github.com/Yoast/wordpress-seo/issues/5614
